x-pack/plugins/security_solution/common/search_strategy/security_solution/hosts/index.ts

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { IEsSearchResponse } from '../../../../../../../src/plugins/data/common';
+import { CloudEcs } from '../../../ecs/cloud';
+import { HostEcs } from '../../../ecs/host';
+
+import {
+  CursorType,
+  Inspect,
+  Maybe,
+  PageInfoPaginated,
+  RequestOptionsPaginated,
+  SortField,
+  TimerangeInput,
+} from '..';
+
+export type HostsQueries = 'host_all' | 'host_details';
```

Comment:
Could we rename host_details into host_overview?
As in x-pack/plugins/security_solution/server/search_strategy/security_solution/factory/hosts/index.ts line 72, the query for host details is actually host overview.

Or I'm also thinking about having an enum that contains all the queries we have in security app,
for example 
```
enum SecuritySolutionQuries = {
  host = 'host',
  hostOverview = 'hostOverview'
}
type HostQueries = SecuritySolutionQuries.host | SecuritySolutionQuries.hostOverview
```
say the same query in host page is reused in network page in the future, we could do
```
type NetworkQueries = SecuritySolutionQuries... | SecuritySolutionQuries.hostOverview
```

Reply:
and in x-pack/plugins/security_solution/public/hosts/containers/hosts/index.tsx Line 69
we could put the enum we defined as well.

Reply:
I think you are right we should use an enum, but I still think we should create the enum of the queries in their respective folder and then combine them back the way we are doing it with `FactoryQueryTypes`